oct/pages/base/base.py

```diff
@@ -37,6 +66,20 @@ def _setup(self):
                             by=find_by, value=locator_value, elcls=element_class)
                         )
 
+        if self.loads:
+            setattr(self, 'elements_loaded', 0)
+
+            locator_name, locator_value = self.loads.get('group locator')
+            element_class = self._check_class(self.loads.get('class'))
+            find_by = getattr(By, locator_name.upper())
+
+            elements = self._reff.find_elements(find_by, locator_value)
+            for index, element in enumerate(elements):
+                attr = ("%s_%d" % (element_class.__name__.lower(), index))
+
+                setattr(self, attr, element_class(element))
```

Comment:
```suggestion
            elements = self._reff.find_elements(find_by, locator_value)
            attr = f"{element_class.__name__.lower()}s"
            setattr(self, attr, [element_class(element) for element in elements])
```

Reply:
Can not apply list comprehension in that way since <attr> is inside the loop and changes its value on each iteration

Reply:
I suggested using one property for the list instead of a property per each item in a list.
instead of "product_thumb_1" and "product_thumb_2" it will be "product_thumbs" that returns a list of product thumb instances 
'attr' isn't in for loop in my suggestion